Update FAQ under personalizing_conda_on_gyre_sverdrup

andrewfagerheim opened the pull request:
I created a bullet for what to do if you've created and activated an environment, but it doesn't show up under Jupyter Hub's list of kernels.

andrewfagerheim committed:
Update FAQ under personalizing_conda_on_gyre_sverdrup

I created a bullet for what to do if you've created and activated an environment, but it doesn't show up under Jupyter Hub's list of kernels.

dhruvbalwada commented:
This is great. Doing this should be similar to saying that `ipykernel` should necessarily be included in the environment.yml package list, right?

andrewfagerheim commented:
Yes, I think that's right

dhruvbalwada commented:
I think the solution should not suggest to manually install ipykernel, but rather to add this item to environment.yml file and then run the conda update using step X in [here](https://github.com/ocean-transport/guides/blob/master/Setting_up_conda_on_clusters.md). 
Could you update the pull request accordingly?

andrewfagerheim committed:
revised FAQ

I made 2 changes:
- added ipykernel to the list of packages in the example environment.yml file
- pointed to the guide on adding packages later, if the original environment.yml file didn't have ipykernel

andrewfagerheim commented:
Just created a new commit with these changes
